src/makeCalculator.js

```diff
@@ -4,7 +4,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculated = {
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+    operate: function (operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - Similarly, for the `operate` method, you should use the method shorthand syntax instead of the function keyword.

BAD EXAMPLE:
operate: function(operation, number) {
  this.result = operation(this.result, number);
  return this;
},

GOOD EXAMPLE:
operate(operation, number) {
  this.result = operation(this.result, number);
  return this;
},